solidity/ecdsa/contracts/WalletRegistryGovernance.sol

```diff
@@ -71,6 +71,9 @@ contract WalletRegistryGovernance is Ownable {
     address payable public newReimbursementPool;
     uint256 public reimbursementPoolChangeInitiated;
 
+    uint256 public notifyOperatorInactivityGasOffsetChangeInitiated;
+    uint256 public newNotifyOperatorInactivityGasOffset;
```

Comment:
Can we place them above `newReimbursementPool`, right after DKG result approval gas offset fields?

Reply:
Sure